### ISSUE TITLE
Narrator does not announce the Domain value "⁅𝑥∈ℝ⁆" when focus navigates there.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1387]
**Pre-Requisites:**
Expressions should be added.

**Repro Steps:** 
1. Launch Calculator app.
2. Turn on the narrator.
3. Navigate Open navigation button and activate it.
4. Navigate to graphing calculator list item and activate it.
5. Navigate to Enter a expression edit box.
6. Enter any expression say "7+8".
7. Open context menu and activate 'Analyze function' button and activate it.
8. Analyze function pane will get open.
9. Now press tab key to navigate to the domain value.
10. Observe the issue.

**Actual Result:**
Narrator does not announce the Domain value "⁅𝑥∈ℝ⁆" when focus navigates there. Narrator just announces "scan off".

**Expected Result:**
Narrator should announce the Domain value "⁅𝑥∈ℝ⁆" when focus navigates there.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
